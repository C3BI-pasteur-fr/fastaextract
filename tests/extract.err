file test.fa: Extracted 3 from 4 sequences (75.00 %)
