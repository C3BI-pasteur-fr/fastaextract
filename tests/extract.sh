#! /bin/sh

### Set verbose mode
test "x$VERBOSE" = "xx" && set -x

#extract without list
../src/faextract -o tmp.out $srcdir/test.fa 2> tmp.err || exit 1
diff $srcdir/test.fa tmp.out || exit 1
diff $srcdir/fullextract.err tmp.err || exit 1

#extract with list
../src/faextract -o tmp.out -l $srcdir/id_list.txt $srcdir/test.fa 2> tmp.err || exit 1
diff $srcdir/extract.out tmp.out || exit 1
diff $srcdir/extract.err tmp.err || exit 1


exit 0

