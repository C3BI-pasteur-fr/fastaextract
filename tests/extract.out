>seq1 nucleic
ggtgttgcgcttgctaccgacgggccagttcgggcaatagggccatctgctacttctgggcagcttgacttcttaggggc
cgttgaaagctgcctgaacgtaatgactggttacaggacaaccctttgaatacgtagctaaaatcacaggtttatacatg
tttcatagacttaccggattcacggggaaaggagctggcc
>seq2 nucleic
GGTGTTGCGCTTGCTACCGACGGGCCAGTTCGGGCAATAGGGCCATCTGCTACTTCTGGGCAGCTTGACTTCTTAGGGGC
CGTTGAAAGCTGCCTGAACGTAATGACTGGTTACAGGACAACCCTTTGAATACGTAGCTAAAATCACAGGTTTATACATG
TTTCATAGACTTACCGGATTCACGGGGAAAGGAGCTGGCC
>seq3 proteic
HTHSGUTGRHVTVCSBWCDDHNTVUDKHMTUHNTHWVCDHYTUMWMDTCVSBBMDARNVYGKTGGMSMUBYKHUMHRRUH
VNKAAWRHMMHACDHKGAKUGVDSVANTATGUNGMABTAT
